docs/administration.platforms.md

```diff
@@ -91,13 +91,15 @@ If you want to remove quotas from a platform just click the "-" button on the ri
 
 ![Manage Platform Quota Definitions](assets/tenants/platform-quota-definitions.png)
 
-### Platform Deactivation
+### Platform Danger Zone
```

Comment:
d: Perhaps better split it into 2 headlines. One for Deprecation and one for Deletion. I think that provides a better overview on what this section is about. Danger Zone leaves room for a lot of interpretation. But reading about deprecation and deletion you directly know what it is about.